docs/docsite/rst/user_guide/playbooks_error_handling.rst

```diff
@@ -93,14 +95,18 @@ In previous version of Ansible, this can still be accomplished as follows::
         msg: "the command failed"
       when: "'FAILED' in command_result.stderr"
 
-You can also combine multiple conditions to specify this behavior as follows::
+You can also combine multiple conditions for failure. This example will fail if both conditions are true::
 
     - name: Check if a file exists in temp and fail task if it does
       command: ls /tmp/this_should_not_be_here
       register: result
       failed_when:
-        - '"No such" not in result.stdout'
         - result.rc == 0
+        - '"No such" not in result.stdout'
+
+ If you want the task to fail when only one condition is satisfied, change the ``failed_when`` definition to::
+
+      failed_when: result.rc == 0 or "No such" not in result.stdout
```

Comment:
probably we also want to add 'multiline' example for those that have very long conditionals

Reply:
I'm going to file that under "future work" and call this PR good enough for now.